src/main/resources/db/migration/V28__iat_ddl.sql

```diff
@@ -0,0 +1,2 @@
+-- Adds a new boolean column to the tims history table to manage if the historical item can be previewed in TIMS
+ALTER TABLE item_history ADD COLUMN "supports_preview" BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
This will be false for any history record created prior to the changes around versioned attachments.  Historical preview will use this to show in the dropdown.